store/storage_manager_builder.go

```diff
@@ -408,14 +409,10 @@ func (smb *StorageManagerBuilder) buildRelayClient(
 // payloads directly from EigenDA validators
 func (smb *StorageManagerBuilder) buildValidatorPayloadRetriever(
 	ethClient common_eigenda.EthClient,
+	ethReader *eth.Reader,
```

Comment:
does the ethReader expose the ethClient methods? If so can we only use the ethReader?

Reply:
iiuc `ethReader` is really just `eigenDASysContractReader` - seems like there could be some benefit to reabstracting this thing